.github/templates/UNUSED_DEPS_ISSSUE.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "chore: some installed deps are not needed"
+labels: automated-issue
+---
+
+Some dependencies specified in `Cargo.toml` are not needed.
+
+Check the [unused dependencies sanity check]({{env.WORKFLOW_URL}}) workflow for details.
+
+This issue was raised by the workflow at `.github/workflows/unused-deps.yml`.
```

Comment:
Nit: Should we include the full URL like in #5, since the issue will be opened in another repo?